### ISSUE TITLE
Add ability to uninstall environments

### ISSUE THREAD

**Issue by pavelzw** [prefix-dev/pixi#749]
### Problem description

It would be nice to be able to uninstall environments that you don't want to have anymore without executing `rm -rf .pixi/envs/<env-name>`.

Maybe something like `pixi uninstall-env <env-name>`? I am not a super-fan of the command name but cannot think of a better one.

**Comment by ruben-arts** [comment]
I guess that would be a `clean` command. 

``` 
pixi clean # e.g. rm -r .pixi
pixi clean -e lint # e.g. rm -r .pixie/envs/lint
pixi clean --cache # e.g. rm -r $PIXI_CACHE_DIR
```


But I'm not convinced yet. As long as it has no autocompletion it is probably less convenient than the `rm -r` command.

**Comment by pavelzw** [comment]
I like `clean` :) 

For people who don't know where the default `$PIXI_CACHE_DIR` is located this might be interesting.

**Comment by ruben-arts** [comment]
Closed as fixed in #1325
